custom/icds_reports/reports/awc_infrastracture.py

```diff
@@ -157,8 +160,52 @@ def get_data_for(month, filters):
                     'format': 'percent_and_div',
                     'frequency': 'month',
                     'redirect': 'awc_infrastructure/medicine_kit'
-                }
+                },
+                {
+                    'label': _('AWCs Reported Infantometer'),
```

Comment:
Hey this is returning the infantometer and stadiometer result without feature flag. Is it prevented in the UI to not show them ? @challabeehyv

Reply:
@Rohit25negi  i handled it in the UI

Reply:
@Rohit25negi as per the design doc, stadiometer is in a new line and infantometer is in line with medical kit.